2.4/alpine/Dockerfile

```diff
@@ -203,9 +203,13 @@ RUN set -eux; \
 		-e 's!^(\s*CustomLog)\s+\S+!\1 /proc/self/fd/1!g' \
 		-e 's!^(\s*ErrorLog)\s+\S+!\1 /proc/self/fd/2!g' \
 		-e 's!^(\s*TransferLog)\s+\S+!\1 /proc/self/fd/1!g' \
+    	-e 's!^(\s*User)\s+daemon\s*$!\1 www-data!g' \
+		-e 's!^(\s*Group)\s+daemon\s*$!\1 www-data!g' \
```

Comment:
Whitespace is a bit off here :eyes:

Reply:
Fixed!

Reply:
Should be fixed.

Reply:
Sorry, you were right. It wasn't 🤦 . Now it should :)